lib/fizzy/parser_expr.cpp

```diff
@@ -560,9 +568,16 @@ parser_result<Code> parse_expr(
         case Instr::local_set:
         case Instr::local_tee:
         {
-            uint32_t imm;
-            std::tie(imm, pos) = leb128u_decode<uint32_t>(pos, end);
-            push(code.immediates, imm);
+            uint32_t idx;
+            std::tie(idx, pos) = leb128u_decode<uint32_t>(pos, end);
+
+            if (instr == Instr::local_get || instr == Instr::local_set || instr == Instr::local_tee)
```

Comment:
This is always true.

Reply:
Right, will remove.

It is a remnant from before #351 was meged.

Reply:
Fixed.